dev-lang/mrustc/mrustc-0.11.2.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 2024-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multiprocessing rust-toolchain
+
+# The makefile needs to know the version of rust to build
+RUST_VERSION=1.74.1
+
+DESCRIPTION="Mutabah's Rust Compiler"
+HOMEPAGE="https://github.com/thepowersgang/mrustc"
+
+if [[ ${PV} == *"9999"* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/thepowersgang/mrustc.git"
+else
+	SRC_URI="https://github.com/thepowersgang/mrustc/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+		https://static.rust-lang.org/dist/rustc-${RUST_VERSION}-src.tar.xz
+	"
+	KEYWORDS="amd64"
```

Comment:
~amd64? We should consider p.use.stable.mask-ing the new USE too at first, just to let issues get shaken out.